application/listeners/posthelpers.go

```diff
@@ -6,83 +6,166 @@ import (
 	"github.com/silinternational/wecarry-api/notifications"
 )
 
-type PostMsgRecipient struct {
-	nickname string
-	email    string
+// The intention is to allow the tests to override this with their own Send function
+var ntfSend = notifications.Send
+
+type PostUser struct {
+	Nickname string
+	Email    string
+}
+
+type PostUsers struct {
+	Requester PostUser
+	Provider  PostUser
 }
```

Comment:
I'm exporting them so that they are available to the code that calls the exported function `GetPostUsers`.  Combining them is fine.  We should probably use a single function that does the combining.